ruby/vampires.rb

```diff
@@ -30,6 +32,7 @@ def vampire_test
 
 		puts "would you like to enroll in the company's health insurance? (y/n)"
 			wants_health_insurance = gets.chomp
+
 				if wants_health_insurance == "y"
 					wants_health_insurance = true
 				else 
```

Comment:
On line 69 consider this

with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`